src/main/kotlin/com/openlattice/assembler/Assembler.kt

```diff
@@ -478,24 +487,16 @@ class Assembler(
         }
     }
 
-
     class OrganizationAssembliesInitializerTask : HazelcastInitializationTask<Assembler> {
         override fun getInitialDelay(): Long {
             return 0
         }
 
         override fun initialize(dependencies: Assembler) {
             dependencies.acm.createRenameDatabaseFunctionIfNotExists()
-            val currentOrganizations =
-                    dependencies
-                            .securableObjectTypes.keySet(
-                                    Predicates.equal(
-                                            "this"
-                                            , SecurableObjectType.Organization
-                                    )
-                            )
-                            .map { it.first() }
-                            .toSet()
+            val currentOrganizations = dependencies.securableObjectTypes.keySet(
```

Comment:
seems like this would be simpler/faster by just taking the keySet of the ORGANIZATIONS map?

Reply:
that sounds good but I'd rather wait until the AssemblerConnectionManager shredding happens in full as this functionality would likely move somewhere that has more intentional access to the orgs map